From 893d3c8ab78d51abeee462e8ab733dad4df3ca48 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jakab=20Krist=C3=B3f?= <jakab.kristof@balasys.hu>
Date: Thu, 14 Jul 2016 08:37:42 +0200
Subject: [PATCH 14/19] Revert "UBUNTU: SAUCE: UEFI: asus-wmi: Restrict debugfs
 interface when module loading is restricted"

This reverts commit 7578cf0d90bf428bf4068c3e519e8f58333b8b4b.
---
 drivers/platform/x86/asus-wmi.c | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/drivers/platform/x86/asus-wmi.c b/drivers/platform/x86/asus-wmi.c
index 071171b..efbc3f0 100644
--- a/drivers/platform/x86/asus-wmi.c
+++ b/drivers/platform/x86/asus-wmi.c
@@ -1868,9 +1868,6 @@ static int show_dsts(struct seq_file *m, void *data)
 	int err;
 	u32 retval = -1;
 
-	if (secure_modules())
-		return -EPERM;
-
 	err = asus_wmi_get_devstate(asus, asus->debug.dev_id, &retval);
 
 	if (err < 0)
@@ -1887,9 +1884,6 @@ static int show_devs(struct seq_file *m, void *data)
 	int err;
 	u32 retval = -1;
 
-	if (secure_modules())
-		return -EPERM;
-
 	err = asus_wmi_set_devstate(asus->debug.dev_id, asus->debug.ctrl_param,
 				    &retval);
 
@@ -1914,9 +1908,6 @@ static int show_call(struct seq_file *m, void *data)
 	union acpi_object *obj;
 	acpi_status status;
 
-	if (secure_modules())
-		return -EPERM;
-
 	status = wmi_evaluate_method(ASUS_WMI_MGMT_GUID,
 				     1, asus->debug.method_id,
 				     &input, &output);
-- 
2.9.2

